modular_skyrat/modules/organs/code/tongue.dm

```diff
@@ -19,3 +19,19 @@
 	speaker.verb_whisper = initial(verb_whisper)
 	speaker.verb_sing = initial(verb_sing)
 	speaker.verb_yell = initial(verb_yell)
+
+/// This "human" tongue is only used in Character Preferences / Augmentation menu.
+/// The base tongue class lacked a say_mod. With say_mod included it makes a non-Human user sound like a Human.
+/obj/item/organ/internal/tongue/human
+	say_mod = "says"
```

Comment:
Don't change it, it doesn't have a say_mod on purpose, so that races and the like can have theirs instead.

This is bad, as this is the default tongue option for everyone.

Reply:
As far as I can tell, it is not the default option. `/datum/augment_item/organ/tongue` is the default which allows species `say_mod` to work, whereas `/obj/item/organ/internal/tongue/human` would be selected in cases where you'd want to override that variable.

That said, is it still an issue even if it's not the default?

Reply:
Ok, I have adjusted this PR with these considerations in-mind.

I have now re-named it to Human Tongue so it's a more clear distinction, and more obvious that the say verb will change. I also re-named the default organ Augment option from "Organic" to "Default" because it makes more sense in the GUI, especially for synthetic/robotic characters.